src/material-experimental/mdc-button/button.scss

```diff
@@ -69,7 +69,7 @@
 // However, Angular Material expects a `mat-icon` instead. The following
 // mixins will style the icons appropriately.
 .mat-mdc-button {
-  .mat-icon {
+  & > .mat-icon {
```

Comment:
Do we need the same change in the `fab.scss`?

Reply:
Yes, I guess for extended fabs that is theoretically something that could happen